src/components/interface/SwapForm/components/Leverage/Leverage.tsx

```diff
@@ -123,57 +90,31 @@ const Leverage = ({
       <Box
         sx={{
```

Comment:
you can move this style object to the style.ts file. Keep the logic component separate from styling

Reply:
Moved it to the style file